CommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue

```diff
@@ -0,0 +1,40 @@
+alts:
+- vegeta
+-vegetaSSB+
```

Comment:
this is mis-spaced. 
Also, why are there caps? 
Also, why is SSB+ an alt of beyond blue?
Also, where are his other transformations?